components/script/dom/globalscope.rs

```diff
@@ -458,13 +459,9 @@ pub(crate) struct ManagedMessagePort {
     /// Whether the port has been closed by script in this global,
     /// so it can be removed.
     explicitly_closed: bool,
-    /// Note: it may seem strange to use a pair of options, versus for example an enum.
-    /// But it looks like tranform streams will require both of those in their transfer.
-    /// This will be resolved when we reach that point of the implementation.
-    /// <https://streams.spec.whatwg.org/#abstract-opdef-setupcrossrealmtransformreadable>
-    cross_realm_transform_readable: Option<CrossRealmTransformReadable>,
-    /// <https://streams.spec.whatwg.org/#abstract-opdef-setupcrossrealmtransformwritable>
-    cross_realm_transform_writable: Option<CrossRealmTransformWritable>,
+    /// Handler for a transfered transform stream.
+    /// A message port can be used to transfer either a readable or a writable stream.
```

Comment:
This is incorrect; rather this is used as the handler for the port in either setup cross realm readable or writable(see for example https://streams.spec.whatwg.org/#abstract-opdef-setupcrossrealmtransformwritable), which is used for the transfer of all types of streams.